docs/content/config/advanced/podman.md

```diff
@@ -96,6 +96,107 @@ docker compose up -d mailserver
+
+    [`podman generate systemd`][podman-docs::cli::generate-systemd] has been deprecated in favor of Quadlets (_since Podman v4.4_).
+
+!!! info "What is a Quadlet?"
+
+    A [Quadlet][podman::quadlet::introduction] file uses the [systemd config format](https://www.freedesktop.org/software/systemd/man/latest/systemd.syntax.html) which is similar to the INI format.
+
+    [Quadlets define your podman configuration][podman-docs::quadlet::example-configs] (_pods, volumes, networks, images, etc_) which are [adapted into the equivalent systemd service config files][podman::quadlet::generated-output-example] at [boot or when reloading the systemd daemon][podman-docs::config::quadlet-generation] (`systemctl daemon-reload` / `systemctl --user daemon-reload`).
+
+[podman-docs::cli::generate-systemd]: https://docs.podman.io/en/latest/markdown/podman-generate-systemd.1.html
+[podman-docs::quadlet::example-configs]: https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#examples
+[podman-docs::config::quadlet-generation]: https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#description
+[podman::quadlet::introduction]: https://mo8it.com/blog/quadlet/
+[podman::quadlet::generated-output-example]: https://blog.while-true-do.io/podman-quadlets/#writing-quadlets
+
+!!! tip "Rootless compatibility"
+
+    Quadlets can [support rootless with a few differences][podman::rootless-differences]:
+
+    - `Network=pasta` configures [`pasta`][network-driver::pasta] as a rootless compatible network driver (_a popular alternative to `slirp4netns`. `pasta` is the default for rootless since Podman v5_).
+    - `Restart=always` will auto-start your Quadlet at login, rootless support requires to enable [lingering][systemd-docs::loginctl::linger] for your user:
+
+        ```bash
+        loginctl enable-linger user
+        ```
+    - [Config locations between rootful vs rootless][podman-docs::quadlet::config-search-path].
+
+[podman::rootless-differences]: https://matduggan.com/replace-compose-with-quadlet/#rootless
+[network-driver::pasta]: https://passt.top/passt/about/#pasta
+[systemd-docs::loginctl::linger]: https://www.freedesktop.org/software/systemd/man/latest/loginctl.html#enable-linger%20USER%E2%80%A6
+[podman-docs::quadlet::config-search-path]: https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#podman-rootful-unit-search-path
+
+
+#### Example Quadlet file
+
+We have to use the .container extension for the quadlet generator to pick up the service.
+Because docker-mailserver uses multiple users inside the container, we will either have to use our own user as root, resulting in our e-mails being owned by a subuid. Alternatively, using UIDMap we can map our rootless user to UID 5000 in the container who owns our e-mails. Using UIDMap also maps root user 0 inside the container to an available sub-uid of our rootless user. Otherwise the container will not have permission to configure itself.
```

Comment:
The choice is arbitrary in the end. I wanted the mail-storage mount to be owned by the rootless user starting the container, and have root inside the container not directly mapped to the user starting the container.

I spent a long time too messing around with uid-maps, and it solves a lot of problems that rootless podman-containers seem to have for beginners. I seem to have learned even more from your findings, thanks!

Reply:
> The choice is arbitrary in the end.

Yes, agreed. Just curious why you might have preferred this choice.

> I wanted the mail-storage mount to be owned by the rootless user starting the container, and have root inside the container not directly mapped to the user starting the container.

Ok, but no practical reason for that right? Since you'd rarely need access to this content outside of the container (_beyond persistent storage_) and you're more likely as a user to want to have ownership of `root` files for the Config Volume yeah? (_similar benefit for logs too_).

`idmap` option for `Volume` settings would be useful for such, but with rootless it's just not available unfortunately.

If you did have a legitimate use-case for preferring `5000` mapped to your host user that would be good to know. I'll otherwise switch to `UIDMap=+0:@%U` for the official advice, with some context added to the docs that I've touched on above and in my other review feedback, with `UIDMap=+5000:@%U` as an alternative example.

> I seem to have learned even more from your findings, thanks!

You're welcome!

---

I'll try allocate some time tomorrow for the final revision and we can then get this merged! 😁